bigbluebutton-html5/imports/api/meetings/server/modifiers/addMeeting.js

```diff
@@ -66,6 +66,8 @@ export default function addMeeting(meeting) {
     metadataProp: Object,
   });
 
+  const newMeeting = meeting;
```

Comment:
Not sure if you need this copy. 
`meeting.welcomeProp.welcomeMsg = meeting.welcomeProp.welcomeMsg.replace(................')` could work too?

Reply:
I think he did it this way to avoid a lint error.
`Disallow Reassignment of Function Parameters (no-param-reassign)`

Reply:
I guess the right way to say it is Bobak wanted to avoid the `Reassignment of Function Parameters ` :+1: Makes sense now, tnx Kert